libavcodec/vaapi_decode.c

```diff
@@ -577,10 +577,10 @@ static int vaapi_decode_make_config(AVCodecContext *avctx,
         switch (avctx->codec_id) {
         case AV_CODEC_ID_H264:
         case AV_CODEC_ID_HEVC:
+        case AV_CODEC_ID_AV1:
```

Comment:
will this impact dxva?
why it will leak surfaces?
It's better to move this commit earlier than the previous one, else film grain clips will have an issue with https://github.com/intel-media-ci/ffmpeg/pull/412/commits/db2cf91690316c44cf16e93931979a9f9e0d18a3

Reply:
Will modify, thanks.